LGTM C# Failing

Follow on of #429 

```
[2020-06-13 12:35:05] [build] (CoreCompile target) -> 
[2020-06-13 12:35:05] [build]   AudioControllerTests.cs(8,28): error CS0234: The type or namespace name 'Http' does not exist in the namespace 'Microsoft.AspNetCore' (are you missing an assembly reference?) [/opt/src/Backend.Tests/Backend.Tests.csproj]
[2020-06-13 12:35:05] [build]   AvatarControllerTests.cs(6,28): error CS0234: The type or namespace name 'Http' does not exist in the namespace 'Microsoft.AspNetCore' (are you missing an assembly reference?) [/opt/src/Backend.Tests/Backend.Tests.csproj]
[2020-06-13 12:35:05] [build]   LiftControllerTests.cs(11,28): error CS0234: The type or namespace name 'Http' does not exist in the namespace 'Microsoft.AspNetCore' (are you missing an assembly reference?) [/opt/src/Backend.Tests/Backend.Tests.csproj]
[2020-06-13 12:35:05] [build]   PermissionServiceMock.cs(3,28): error CS0234: The type or namespace name 'Http' does not exist in the namespace 'Microsoft.AspNetCore' (are you missing an assembly reference?) [/opt/src/Backend.Tests/Backend.Tests.csproj]
[2020-06-13 12:35:05] [build]   ProjectControllerTests.cs(6,28): error CS0234: The type or namespace name 'Http' does not exist in the namespace 'Microsoft.AspNetCore' (are you missing an assembly reference?) [/opt/src/Backend.Tests/Backend.Tests.csproj]
[2020-06-13 12:35:05] [build]   UserEditControllerTests.cs(7,28): error CS0234: The type or namespace name 'Http' does not exist in the namespace 'Microsoft.AspNetCore' (are you missing an assembly reference?) [/opt/src/Backend.Tests/Backend.Tests.csproj]
[2020-06-13 12:35:05] [build]   PermissionServiceMock.cs(9,40): error CS0246: The type or namespace name 'HttpContext' could not be found (are you missing a using directive or an assembly reference?) [/opt/src/Backend.Tests/Backend.Tests.csproj]
[2020-06-13 12:35:05] [build]   PermissionServiceMock.cs(14,65): error CS0246: The type or namespace name 'HttpContext' could not be found (are you missing a using directive or an assembly reference?) [/opt/src/Backend.Tests/Backend.Tests.csproj]
[2020-06-13 12:35:05] [build]   PermissionServiceMock.cs(19,37): error CS0246: The type or namespace name 'HttpContext' could not be found (are you missing a using directive or an assembly reference?) [/opt/src/Backend.Tests/Backend.Tests.csproj]
[2020-06-13 12:35:05] [build]   PermissionServiceMock.cs(24,33): error CS0246: The type or namespace name 'HttpContext' could not be found (are you missing a using directive or an assembly reference?) [/opt/src/Backend.Tests/Backend.Tests.csproj]
[2020-06-13 12:35:05] [build]   PermissionServiceMock.cs(7,35): error CS0012: The type 'HttpContext' is defined in an assembly that is not referenced. You must add a reference to assembly 'Microsoft.AspNetCore.Http.Abstractions, Version=3.1.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60'. [/opt/src/Backend.Tests/Backend.Tests.csproj]
[2020-06-13 12:35:05] [build]     0 Warning(s)
[2020-06-13 12:35:05] [build]     11 Error(s)
```

--- a/Backend.Tests/Backend.Tests.csproj
+++ b/Backend.Tests/Backend.Tests.csproj
@@ -1,4 +1,4 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
     <TargetFramework>netcoreapp3.1</TargetFramework>

